test/cloudant.js

```diff
@@ -77,7 +77,7 @@ describe('Cloudant #db', function() {
         .get(`/_api/v2/db/${DBNAME}/_security`)
         .reply(200, security);
 
-      var cloudant = Cloudant({ url: SERVER, username: ME, password: PASSWORD });
+      var cloudant = Cloudant({ url: SERVER, username: ME, password: PASSWORD, plugins: [] });
```

Comment:
As per my comment on the other test, disabling the cookie auth plugin forces all of our service tests to run using basic auth. I'm opposed to that given that using cookie auth is the default. I understand that the changes to the cookie model to return errors has made a change necessary here to accommodate mock testing that doesn't actually honour the cookie requests so I propose:
`plugins: []` -> `plugins: testPlugins`
where `testPlugins = (process.env.NOCK_OFF) ? [cookieauth] : []` or equivalent

wdyt?

Reply:
I think if this works as a solution we can avoid the session mocks for the readme example tests too. Sorry I didn't register earlier how pervasive the problem was and that this might be an option.